apm-lambda-extension/extension/route_handlers.go

```diff
@@ -80,12 +84,8 @@ func handleIntakeV2Events(agentDataChan chan AgentData) func(w http.ResponseWrit
 		w.WriteHeader(http.StatusAccepted)
 		w.Write([]byte("ok"))
 
-		if r.Body == nil {
```

Comment:
Presuming the "r.Body will always be `nil`" docs are this line here: https://github.com/golang/go/blob/18934e11ba6ef2b2f21f091ddf4ab6814dcf1959/src/net/http/request.go#L174